Added pairs sizing command

def-mycroft opened the pull request:
Added functionality for calculating volatility-weighted unit size for a
pairs trade. Folder `src/pairs` was created and module `sizing` was
added, along with supporting `commands` module.

Added test case for `sizing.py`.

Made minor edits to README.

Added line `*.png` to `.gitignore` to suppress png files from git
tracking.

def-mycroft committed:
Added pairs sizing command

Added functionality for calculating volatility-weighted unit size for a
pairs trade. Folder `src/pairs` was created and module `sizing` was
added, along with supporting `commands` module.

Added test case for `sizing.py`.

Made minor edits to README.

Added line `*.png` to `.gitignore` to suppress png files from git
tracking.

Graeme22 commented on src/pairs/sizing.py:
```diff
@@ -0,0 +1,117 @@
+        (vol_right * price_right * multiplier_right)
+
+    msg = 'Volatility-weighted equity should equate.'
+    assert vol_left * price_left * multiplier_left * unit_size_left == \
```
Can this be moved to a test? We probably don't want asserts in production code.

Graeme22 replied:
Plus, this is vulnerable to small rounding errors.

Graeme22 commented:
Looks good as a start. Just one comment above.

def-mycroft committed:
Removed assertion from sizing.py

Assertion doesn't really need to be present in production code and is
vulnerable to small rounding errors.

Graeme22 reviewed (approved):
Looks good!
